src/NuGet/BuildNuGets.csx

```diff
@@ -87,6 +87,7 @@ string[] RedistPackageNames = {
     "Microsoft.CodeAnalysis.EditorFeatures",
     "Microsoft.CodeAnalysis.EditorFeatures.Text",
     "Microsoft.CodeAnalysis.Features",
+    "Microsoft.CodeAnalysis.Remote.Razor.ServiceHub",
```

Comment:
I agree, I think the goal is that these would only be on myget because there's no reason for an end user to use these packages. 

Is `NonRedistPackageNames` or `PreReleaseOnlyPackages` the right home?